tests/versioning_3_test.go

```diff
@@ -878,6 +877,403 @@ func (s *Versioning3Suite) testIndependentActivity(behavior enumspb.VersioningBe
 	s.verifyWorkflowVersioning(tvWf, behavior, tvWf.Deployment(), nil, nil)
 }
 
+func (s *Versioning3Suite) TestChildWorkflowInheritance_PinnedParent() {
+	s.testChildWorkflowInheritance_ExpectInherit(false, false, vbPinned)
```

Comment:
```suggestion
        // same task queue, no override, parent is pinned -> expect child to start on parent's pinned version
	s.testChildWorkflowInheritance_ExpectInherit(false, false, vbPinned)
```
Writing this partially to confirm we're on the same page, partially because I do think little comments here like this would make it easier for future readers (maybe us in a year) to quickly understand this